### DISCUSSION TITLE
RootTagMissingFromViewException when no HTML for a Component

### DISCUSSION THREAD

**Discussion by AdamEsterle** [livewire/livewire#6706] [7. Bugs (archived)]
### Livewire version

3

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Include a component
`<livewire:impersonation-banner />`
2. If component has an if statement that is false, is throws RootTagMissingFromViewException since no HTML is rendered
```
@if($shouldShow) // exception if $shouldShow = false
    <div class="font-medium text-white">
                    You are impersonating {{ auth()->user()->name }}.
     </div>
@endif
```
3. Worked fine in 2.x, works in 3.x if $shouldShow = true

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@AdamEsterle Livewire requires a root element, so you should still wrap your `@if` in a `<div>`.
